odl/space/cu_ntuples.py

```diff
@@ -536,19 +536,24 @@ def __init__(self, size, dtype, **kwargs):
 
             Only scalar data types are allowed.
 
-        weight : `array-like`, `CudaFnVector` or `float`, optional
-            Use weighted inner product, norm, and dist.
+        weight : optional
+            Use weighted inner product, norm, and dist. The following
+            types are supported as ``weight``:
 
-            `float`:
+            `FnWeightingBase` :
```

Comment:
Yeah it's already the second level, so we're explaining the behavior for different types of input here.